.circleci/config.yml

```diff
@@ -4,17 +4,49 @@ orbs:
   azure-cli: circleci/azure-cli@1.0.0
 
 jobs:
-  build:
+
+  build-11:
+    docker:
+      - image: 'heisenberg302/extbuilder:11.9'
+    steps:
+      - checkout
+      - run:
+          name: 'Configure, Build, and Install'
+          command: build-ext
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build-11/*
```

Comment:
if we checkout at every stage, does this still need to be in the workspace?

Reply:
Yeah because we use this folder for running the tests:

https://github.com/citusdata/the-process/blob/master/circleci/images/exttester/files/sbin/install-and-test-ext#L17

Reply:
Is there a technical reason we can't run from the checkout?

Reply:
technically I think we should be able to do that. As far as I remember Makefile in build-X is already a symbolic link to src/.../Makefile so I am guessing that it should work but that is something to try. I would suggest trying that when updating pg-13 image from 13beta3 to 13.0, what do you think? I can create an issue for trying that on the-process repo.

Reply:
https://github.com/citusdata/the-process/issues/41